variables.tf

```
@@ -51,8 +64,25 @@ variable "query_log_retention" {
   default     = 7
 }
 
+variable "vpc_id" {
+  type = object({
+    vpc_id = optional(string)
+  })
+  description = "VPC ID"
+  default = {
+    vpc_id = null
+  }
+}
```

Adjusting this to match the submodule suggestion:

```suggestion
variable "vpc_id" {
  description = "VPC ID"
  type = object({
    vpc_id = string
  })
  default = null
}
```